app/domain/object/status.go

```diff
@@ -0,0 +1,17 @@
+package object
+
+import (
+	"time"
+)
+
+type Status struct {
+	ID int64 `json:"id,omitempty" db:"id"`
+
+	AccountID int64 `json:"account_id,omitempty" db:"account_id"`
```

Comment:
タグの `json` の部分は今回レスポンスのJSON出力を制御するために設定していますが、レスポンスに `AccountID` を出すことはないので完全に出力しない設定でもいいと思います
```suggestion
	AccountID int64 `json:"-" db:"account_id"`
```